.travis.yml

```diff
@@ -12,6 +12,9 @@ python:
  - "3.4"
  - "pypy"
 install:
+ - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install -q python-memcached>=1.57; fi
+ - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then pip install -q python3-memcached>=1.51; fi
```

Comment:
As of 1.57, python-memcached should run on Python 3, and [it's tested on it](https://travis-ci.org/linsomniac/python-memcached), so is there another reason not to use python-memcached==1.57 for all three cases?